webapp/src/blocks.tsx

```diff
@@ -1392,6 +1392,7 @@ export class Editor extends toolboxeditor.ToolboxEditor {
     }
 
     private filterBlocks(subns: string, blocks: toolbox.BlockDefinition[]) {
+        if (!blocks) return [];
```

Comment:
question - is there a known case where blocks is not set that we're already hitting? This change seems fine but just want to make sure it's not suppressing a real subtle issue (or does this just end up falling to a path of "filterBlocks returns empty so just return all blocks")

Reply:
no I think was only hit with the new changes